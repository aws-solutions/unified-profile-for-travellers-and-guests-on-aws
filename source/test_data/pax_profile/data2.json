{"modelVersion":"1.0","id":"5605582450","lastUpdatedOn":"2021-12-06T05:47:37.238371435Z","createdOn":"2021-08-06T05:47:37.238371435Z","lastUpdatedBy":"Kaylie Hirthe","createdBy":"Marietta Nolan","emails":[{"type":"personal","address":"marquesmetz@hickle.biz","primary":false},{"type":"","address":"chasejones@casper.info","primary":false},{"type":"","address":"curtistowne@sauer.org","primary":true}],"phones":[{"type":"","number":"(350)013-0821","primary":true,"countryCode":84},{"type":"mobile","number":"1-598-934-6817","primary":true,"countryCode":16}],"addresses":[{"type":"shipping","line1":"3009 Course furt","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"San Francisco","state":{"code":"","name":""},"province":{"code":"BC","name":"British Columbia"},"postalCode":"75848","country":{"code":"BJ","name":""},"primary":false}],"honorific":"Mr","firstName":"Michele","middleName":"Derek","lastName":"Mills","gender":"other","pronoun":"he","dateOfBirth":"1934-10-11","language":{"code":"fi","name":""},"nationality":{"code":"CG","name":""},"jobTitle":"Executive","parentCompany":"Morningstar, Inc.","loyaltyPrograms":[{"id":"7571973033","programName":"ascend","miles":356414.0,"milesToNextLevel":9745.0,"level":"platinium","joined":"2021-08-06T05:47:37.238371435Z"}],"identityProofs":null}