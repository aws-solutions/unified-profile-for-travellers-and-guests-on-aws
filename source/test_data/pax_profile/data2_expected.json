{"air_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "id": "8924443001", "last_updated": "2022-02-05T02:54:30.557043995Z", "created_on": "2021-12-05T02:54:30.557043995Z", "last_updated_by": "Clovis Hermann", "created_by": "Amya Labadie", "honorific": "Mrs", "first_name": "Ludwig", "middle_name": "Cordelia", "last_name": "Blanda", "gender": "female", "pronoun": "she", "date_of_birth": "1979-01-02", "job_title": "Designer", "company": "Barchart", "nationality_code": "HN", "nationality_name": "", "language_code": "om", "language_name": "", "email_business": "", "email": "", "email_type": "", "phone": "5327672053", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/02/05/02/54/30/557043995Z", "address_line1": "365 Lakes ton", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Birmingham", "address_state_province": "", "address_postal_code": "65579", "address_country": "SS", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": ""}], "common_email_recs": [], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-02-05T02:54:30.557043995Z", "last_updated_by": "Clovis Hermann", "number": "5327672053", "country_code": 32, "type": "personal", "traveller_id": "8924443001", "last_updated_partition": "2022/02/05/02/54/30/557043995Z"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-02-05T02:54:30.557043995Z", "last_updated_by": "Clovis Hermann", "number": "858.432.5280", "country_code": 10, "type": "mobile", "traveller_id": "8924443001", "last_updated_partition": "2022/02/05/02/54/30/557043995Z"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-02-05T02:54:30.557043995Z", "last_updated_by": "Clovis Hermann", "id": "0518206991", "program_name": "ascend", "miles": 42587.0, "miles_to_next_level": 1667.0, "level": "silver", "joined": "2021-12-05T02:54:30.557043995Z", "traveller_id": "8924443001", "last_updated_partition": "2022/02/05/02/54/30/557043995Z"}, {"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-02-05T02:54:30.557043995Z", "last_updated_by": "Clovis Hermann", "id": "2447481825", "program_name": "elite", "miles": 945120.0, "miles_to_next_level": 6002.0, "level": "platinium", "joined": "2021-12-05T02:54:30.557043995Z", "traveller_id": "8924443001", "last_updated_partition": "2022/02/05/02/54/30/557043995Z"}]}